canonical-websites/www.canonical.com
# Remove dupe dependencies; upgrade cookie-policy and global-nav

## Done

I noticed renovate flagging duplicate dependencies between "dependencies" and "devDependencies" - so I deduped.

## QA

1. Check out this feature branch
2. Run the site using the command `./run`
3. View the site locally in your web browser at: [http://0.0.0.0:8002/](http://0.0.0.0:8002/)
4. Test command `./run build`
5. Test command `./run clean`
6. 4. Test command `./run test`
7. Verify no errors.
